ActorsLeakSpec: must not leak actors

ErrorFilter picking up an excess TimeoutException:

```
[info] - must not leak actors *** FAILED *** (12 seconds, 311 milliseconds)
[info]   java.lang.AssertionError: received 1 excess messages on ErrorFilter(class java.util.concurrent.TimeoutException,None,Left(),false)
[info]   at akka.testkit.EventFilter.intercept(TestEventListener.scala:124)
[info]   at akka.remote.classic.ActorsLeakSpec.f$proxy1$10(ActorsLeakSpec.scala:214)
[info]   at akka.remote.classic.ActorsLeakSpec.$init$$$anonfun$2$$anonfun$1(ActorsLeakSpec.scala:78)
```

--- a/akka-remote/src/test/scala/akka/remote/classic/ActorsLeakSpec.scala
+++ b/akka-remote/src/test/scala/akka/remote/classic/ActorsLeakSpec.scala
@@ -4,8 +4,6 @@
 
 package akka.remote.classic
 
-import java.util.concurrent.TimeoutException
-
 import scala.collection.immutable
 import scala.concurrent.Await
 import scala.concurrent.duration._
@@ -210,9 +208,6 @@ class ActorsLeakSpec extends AkkaSpec(ActorsLeakSpec.config) with ImplicitSender
       }
 
       Await.result(remoteSystem.whenTerminated, 10.seconds)
-
-      EventFilter[TimeoutException](occurrences = 1).intercept {}
-
       awaitAssert(assertResult(initialActors)(targets.flatMap(collectLiveActors).toSet), 10.seconds)
     }
 

